### DISCUSSION TITLE
0.0.15 Beta Release

### DISCUSSION THREAD

**Discussion by Patrick762** [Patrick762/hassio-bluetti-bt#42] [Ideas]
This beta release contains a possible fix for Bluetti devices which have "PBOX" in their Bluetooth name.

If you don't see your device in the integration setup, you should use this version.

This fix will be in the next release after the EP760 has been tested.

<hr /><em>This discussion was created from the release <a href='https://github.com/Patrick762/hassio-bluetti-bt/releases/tag/0.0.15'>0.0.15 Beta Release</a>.</em>

**Comment by brakster2** [comment]
hey Patric,

 can you please add the function for changing the UPS or PV priority to the integration for the AC300 and AC500?

THX

Greets Andy

**Reply by Patrick762** [reply]
I can look into it, but I can't test it. I already contacted Bluetti to maybe get more Info on the Bluetooth protocol, but I didn't get an answer yet.

**Reply by brakster2** [reply]
hey Patrick thanks for your answer. I have a screenshot here. Can you update the PV priority function ? I want to be able to choose myself

![inbound8170758718141634956](https://github.com/Patrick762/hassio-bluetti-bt/assets/85498476/d7ce1c60-6c14-426b-a92a-18ffc9ef99fc)


Thank you very much

Best regards

Andy
